app/routes/_index.tsx

```diff
@@ -41,24 +42,25 @@ const iconComponents: { [key: string]: LucideIcon } = {
 	Dignity: Heart,
 };
 
-function GetIcon({
-	category,
-	color,
-	strokeWidth,
-	size,
-}: {
+interface DynamicCategoryIconProps {
 	category: string;
 	color: string;
 	strokeWidth: string;
 	size: string;
-}) {
-	// this is a temporary fix for the category name mismatch in CMS configuration
-	// FIXME: remove this temporary fix once the category name mismatch is resolved in CMS configuration and it's minted again
-	category = category === "Oppurtunity" ? "Opportunity" : category;
+}
 
+const DynamicCategoryIcon: React.FC<DynamicCategoryIconProps> = ({
+	category,
+	color,
+	strokeWidth,
+	size,
+}) => {
 	const CategoryIcon = iconComponents[category];
+	if (!CategoryIcon) {
+		return <AlertCircle size={14} />; // or a placeholder component
```

Comment:
i think the `size` prop can be passed to `AlertCircle` so it doesn't throw off the alignment here
<img width="249" alt="Screenshot 2024-02-07 at 1 33 27 PM" src="https://github.com/VoiceDeck/app/assets/79813968/ae0ab82c-d6f7-4ed6-af11-824aeedbd2ae">

Reply:
I want it to look out of place because its a sign something is wrong